projects/demo/src/modules/components/editor-new/examples/5/index.ts

```diff
@@ -0,0 +1,57 @@
+import {Component, ViewEncapsulation} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {
+    tiptapEditorStyles,
+    TUI_EDITOR_EXTENSIONS,
+    TUI_EDITOR_STYLES,
+    TuiEditorTool,
+} from '@taiga-ui/addon-editor';
+import {TuiDestroyService} from '@taiga-ui/cdk';
+
+@Component({
+    selector: 'tui-editor-example-5',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    providers: [
+        TuiDestroyService,
+        {
+            provide: TUI_EDITOR_STYLES,
+            useValue: tiptapEditorStyles,
+        },
+        {
+            provide: TUI_EDITOR_EXTENSIONS,
+            useValue: [
+                import('@tiptap/starter-kit').then(({default: module}) => module),
+                import('@taiga-ui/addon-editor/extensions/group').then(
+                    ({createGroupExtension}) => createGroupExtension({draggable: false}),
```

Comment:
Should we write other options from `TuiEditorGroupOptions` to demostrate their defaults for developer ?

What do you think ?

Reply:
I'm not sure, because there are only 3 important options, which are shown in two examples

<img width="528" alt="image" src="https://user-images.githubusercontent.com/12021443/181568881-8df1f106-bee8-4495-b437-359eaa2e3a21.png">